desktop-widgets/tab-widgets/TabDiveInformation.ui

```diff
@@ -445,23 +445,27 @@
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="diveInfoWaterDensityLayout">
-
           <item>
-        <widget class="QLabel" name="waterTypeText">
+           <widget class="QLabel" name="waterTypeText">
+           </widget>
+          </item>
+          <item>
+           <widget class="QComboBox" name="waterTypeCombo"/>
+          </item>
+          <item>
+           <widget class="QLabel" name="salinityOverWrittenIcon">
          <property name="text">
-          <string>EN13319</string>
+          <string>!</string>
          </property>
-        </widget>
+           </widget>
```

Comment:
hand editing these files is not for the faint of heart  - and rarely a good idea. Can you explain what you tried to achieve here?

Reply:
Done